stagecraft/apps/datasets/tests/models/test_data_set.py

```diff
@@ -69,6 +72,31 @@ def test_saving_with_new_name_doesnt_change_it(self):
         data_sets = DataSet.objects.filter(name='data_group1_data_type1')
         assert_equal(len(data_sets), 1)
 
+    def test_can_create_dataset_with_owner(self):
+        DataSet.objects.create(
+            data_group=self.data_group1,
+            data_type=self.data_type1)
+        data_set1 = DataSet.objects.get(name='data_group1_data_type1')
+        user, _ = User.objects.get_or_create(
+            email='foobar.lastname@gov.uk')
+        data_set1.save()
+        data_set1.owners.add(user)
+
+        assert_equal('foobar.lastname@gov.uk', data_set1.owners.first().email)
+
+    def test_can_create_dataset_without_owner(self):
+        DataSet.objects.create(
+            data_group=self.data_group1,
+            data_type=self.data_type1)
+        data_set1 = DataSet.objects.get(name='data_group1_data_type1')
+        data_set1.save()
+
+        assert_equal(len(data_set1.owners.all()), 0)
+
+    def test_can_create_dataset_without_owner_in_forms(self):
```

Comment:
It seems like we may be testing some core django behaviour here - if the model definition has blank for a field, then we would expect this parameter to be true - there may not be value in testing the way django is behaving